code/game/objects/items/roulette.dm

```diff
@@ -0,0 +1,72 @@
+#define SPINNING_DURATION 3 SECONDS
+
+/obj/structure/roulette
+	name = "roulette"
+	desc = "Normally used for giving out rewards.. or choosing torture methods."
+	icon = 'icons/obj/roulette.dmi'
+	icon_state = "blue"
+	density = TRUE
+	/// Is it currently spinning?
+	var/spinning = FALSE
+	/// List of options the roulette currently has
+	var/list/options = list("Blue", "Orange", "Green", "Yellow", "Red", "Purple")
+	/// Colors available on the roulette
+	var/list/colors = list("blue", "orange", "green", "yellow", "red", "purple")
```

Comment:
These are practically the same, can't you use one for both uses?

Reply:
How would you suggest it? Because the var/colors is the names used for the sprites, while the var/options list are to be modified by the player.